lib/src/config.rs

```diff
@@ -488,15 +489,15 @@ impl ConfigFile {
         path: impl Into<PathBuf>,
     ) -> Result<Self, ConfigLoadError> {
         let layer = match ConfigLayer::load_from_file(source, path.into()) {
-            Ok(layer) => layer,
+            Ok(layer) => layer.into(),
```

Comment:
No particular reason. Maybe I would thought that `.into()` was good because the type of the `layer` wasn't explicitly spelled here either. Changed to `Arc::new(..)`.